source/common/logger/ur_level.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: MIT
+
+namespace level {
+
+enum class Level {
+    DEBUG,
+    INFO,
+    WARNING,
+    ERROR,
+    QUIET
+};
+
+static const char *to_str(Level level) {
```

Comment:
Make this `constexpr`.
More generally, put everything `constexpr` when you can.